pkg/daemon/update.go

```diff
@@ -656,7 +656,7 @@ func getFileOwnership(file ignv2_2types.File) (int, int, error) {
 }
 
 func (dn *Daemon) atomicallyWriteSSHKey(newUser ignv2_2types.PasswdUser, keys string) error {
-	authKeyPath := filepath.Join(coreUserSSHPath, "authorized_keys")
+	authKeyPath := filepath.Join(coreUserSSHPath, "authorized_keys.d/ignition")
```

Comment:
since we are already path joining.
```
authKeyPath := filepath.Join(coreUserSSHPath, "authorized_keys.d", "ignition")
```

Reply:
thx @rphillips ! just threw this up quickly to get the path confirmed :)

Reply:
done.